include/zephyr/bluetooth/iso.h

```diff
@@ -691,6 +710,14 @@ struct bt_iso_chan_ops {
 	 * channel is disconnected, including when a connection gets
 	 * rejected or when setting security fails.
 	 *
+	 * If the channel was established then it still contain references to the BIS/CIS handles
+	 * and is not completely free'd at this point. The memory of the channel shall not be
+	 * memset to 0 or free'd.
+	 * To avoid any issue it is recommended to use a @ref k_work_submit or similar to not
+	 * overwrite any data while in the callback.
+	 *
+	 * For the above reason it is still possible to use bt_iso_chan_get_info() on the @p chan.
```

Comment:
I didn't really understand what is meant with this section. Could you elaborate a bit on this?

From a controller point of view, the CIS Central handles remain valid after a disconnection, but this is not the case for a peripheral.

Reply:
Redid the text a bit. The main purpose was to document that this callback is similar to the bt_conn.disconnected callback, and the `chan` object is still valid in the host until after this function returns.